resources/service-catalog-addons/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+dependencies:
```

Comment:
please add req file to helm-broker as well

Reply:
Fixed